packages/lesswrong/lib/collections/tags/fragments.ts

```diff
@@ -216,6 +219,29 @@ registerFragment(`
   }
 `);
 
+registerFragment(`
+  fragment ArbitalLinkedPagesFragment on ArbitalLinkedPages {
```

Comment:
Because this is a fragment on a custom graphql type I'd leave a comment pointing to that type definition and explaining what this is doing (it took me a minute)